selftests/unit/utils/archive.py

```diff
@@ -117,6 +117,7 @@ def get_path(*args):
                 os.path.pardir,
                 os.path.pardir,
                 ".data",
+                "test_archive",
```

Comment:
I would keep the naming conventions for data files, therefore I would rename this from `test_archive` to `archive.py.data`